src/constants.py

```diff
@@ -1,7 +1,12 @@
+
+## Apple App Attestation
 app_id = 'L796QSLV3E.ca.bc.gov.BCWallet'
 rp_id_hash_end = 32
 counter_start = 33
 counter_end = 37
 aaguid_start = 37
 aaguid_end = 53
 cred_id_start = 55
+
+## Redis
+auto_expire_nonce = 60 * 60 * 10 # 10 minutes
```

Comment:
Is this milliseconds or seconds? Either way I don't think that's ten minutes. Looks like it's either 36 seconds or ten hours

Reply:
10 hours. it's seconds. Will fix.

Reply:
Fixed.